kinfer/protos/kinfer.proto

```diff
@@ -36,13 +36,32 @@ message JointPositionValue {
     JointPositionUnit unit = 3;
 }
 
+message JointPositionsValue {
+    repeated JointPositionValue values = 1;
+}
+
 message JointPositionsSchema {
     JointPositionUnit unit = 1;
     repeated string joint_names = 2;
 }
 
-message JointPositionsValue {
-    repeated JointPositionValue values = 1;
+
+/* --------------- *
+ * Joint Signs in the parent frame *
+ * CCW is positive, CW is negative *
+ * --------------- */
+
+message JointSignsSchema {
+    repeated string joint_names = 1;
+}
+
+message JointSignValue {
+    string joint_name = 1;
+    int32 sign = 2;
+}
+
+message JointSignsValue {
+    repeated JointSignValue values = 1;
```

Comment:
so this, and joint offsets, shouldn't live here, because they could be different between sim and real (i.e., urdf doesn't match real joints)